accounts/views.py

```diff
@@ -1,9 +1,11 @@
 from django.views.generic import TemplateView, CreateView
 from django.contrib.auth import get_user_model
 from django.urls import reverse_lazy
-
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import(LoginView, LogoutView)
 
 from .forms import SignUpForm
+from .forms import LoginForm
```

Comment:
```suggestion
from .forms import LoginForm, SignUpForm
```
同じところからimportしているものはまとめてしまいましょう。